tests/ui/consts/issue-17718-const-bad-values.rs

```diff
@@ -5,6 +5,7 @@ const C1: &'static mut [usize] = &mut [];
 
 static mut S: usize = 3;
 const C2: &'static mut usize = unsafe { &mut S };
```

Comment:
This test doesn't seem to depend on the sizedness of the integer, it just was from the before-times when Rust used `int`/`uint`, and that got changed to `isize`/`usize`. I believe we can fix the test stderr having variance based on host by simply doing this:
```suggestion
static mut S: i32 = 3;
const C2: &'static mut i32 = unsafe { &mut S };
```

Reply:
Applied